File: dotnet/src/Pactman.NonprofitCheckPlus/Exceptions/PactmanNotFoundException.cs
namespace Pactman.NonprofitCheckPlus.Exceptions
{
    /// <summary>HTTP 404. No matching record.</summary>
    public sealed class PactmanNotFoundException : PactmanApiException
    {
        /// <summary>Initializes the exception from response metadata.</summary>
        /// <param name="message">Human-readable description of the failure.</param>
        /// <param name="init">The metadata the transport collected from the response.</param>
        public PactmanNotFoundException(string message, ApiErrorInit init)
            : base(message, init, ErrorCategory.NotFound)
        {
        }
    }
}

File: java/sdk/src/main/java/org/pactman/nonprofitcheckplus/exceptions/PactmanNotFoundException.java
package org.pactman.nonprofitcheckplus.exceptions;

/** HTTP 404. */
public class PactmanNotFoundException extends PactmanApiException {

    private static final long serialVersionUID = 1L;

    /**
     * Creates the exception.
     *
     * @param message the message to surface.
     * @param init    the response metadata.
     */
    public PactmanNotFoundException(String message, ApiErrorInit init) {
        super(message, init, ErrorCategory.NOT_FOUND);
    }
}
